### PR TITLE
Adukif 75v1

### PR DESCRIPTION
**Pull request by lukaaduashvili**
Added and improved Adukif 75v1

## Description
Custom keyboard with custom pcb. Currently a personal project.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/adukif_75/config.h
Comment (sol) [inline keyboards/adukif_75/config.h:26]: You can move this (and most other definitions in this file) to `info.json`.

**Review by drashna — CHANGES_REQUESTED** [review]
It looks like you have changes to the submodules here.  These need to be reverted before the PR can be approved. 

Rebasing sometimes will fix this. Otherwise, you'd want to do something like this to fix them:
```sh
git checkout master
git submodule foreach 'git log -n1 --oneline'
# remember the first abcdef1234 on the submodules that have an issue
git checkout @@your-branch@@
# for each of the problematic repo's:
cd lib/@@name@@
git checkout abcdef1234
```

File: keyboards/adukif_75/adukif_75.c
Comment (sol) [inline keyboards/adukif_75/adukif_75.c:4]: I think it should be possible to remove this file.

File: keyboards/adukif_75/adukif_75.h
Comment (sol) [inline keyboards/adukif_75/adukif_75.h:31]: I think there is a macro generated from `info.json` that you can use instead.

File: keyboards/adukif_75/config.h
Comment (sol) [inline keyboards/adukif_75/config.h:26]: I think it’s possible to move the definitions in this file to `info.json`.

**Review by sol — CHANGES_REQUESTED** [review]
I’m not a QMK core contributor. So take this into account when reading my Suggestions. I don’t have any authority here.

- I think this can be significantly simplified with the new data driven approach.
- I’m not sure if it is a good idea to update dependencies/submodules as part of the same PR.

File: keyboards/adukif_75/config.h
Comment (drashna) [reply keyboards/adukif_75/config.h:26]: A lot of this can be generated by running `qmk info -f json -kb (name)`.

**Pull request closed without merging**
